tests/integration/test_api.py

```diff
@@ -39,7 +39,7 @@ def test_get_listens(self):
         # This sleep allows for the influx subscriber to take its time in getting
         # the listen submitted from redis and writing it to influx.
         # Removing it causes an empty list of listens to be returned.
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
Rather than having a sleep here, we should check to see if the service we're waiting for is up yet, using something like dockerize. Not critical this second, but would be nice for later.